scss/_settings.scss

```diff
@@ -1,6 +1,8 @@
 //library namespace
 $fd-namespace: fd !default;
 
+@import "./../node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/sass_variables.scss";
```

Comment:
are we taking the sass variables or the css variables? Should this also be passed to the postcss config? I think there's some infrastructure concerns here. 

I'm ok with this as long as someone has checked that the postcss color backups still work as expected.

Reply:
i thought we agreed to pass both(css and scss variables), if this is not the case we can clean it up

Reply:
No that's fine, but did we check this works as expected with postcss?

You'll need to check the docs site as the playground isn't wired up to have the scss run through postcss, see: https://github.com/SAP/fundamental-styles/issues/296